make/release.mk

```diff
@@ -5,30 +5,31 @@
 ## @category Release
 CMREL_KEY ?=
 
-.PHONY: release
+.PHONY: release-artifacts
 # Build all release artifacts which might be run or used locally, except
 # for anything signed.
-release-artifacts: server-binaries cmctl kubectl-cert_manager helm-chart static-manifests all-containers
+release-artifacts: server-binaries cmctl kubectl-cert_manager helm-chart release-container-bundles release-manifests
 
-.PHONY: release-signed
+.PHONY: release-artifacts-signed
 # Same as `release`, except it also signs the Helm chart. Requires CMREL_KEY
 # to be configured.
-release-artifacts-signed: release-artifacts helm-chart-signature
+release-artifacts-signed: release-artifacts
+	$(MAKE) --no-print-directory helm-chart-signature
 
-.PHONY: staged-release
+.PHONY: release
 ## Create a full release ready to be staged, including containers bundled for
 ## distribution. Requires CMREL_KEY to be configured.
 ##
 ## @category Release
-release: release-signed release-manifests release-containers
+release: release-artifacts-signed
 	$(MAKE) --no-print-directory bin/release/metadata.json
```

Comment:
note: I think `release-artifacts` covers building _everything_, so this target just becomes "release-artifacts" + `metadata.json`